src/compile/layoutsize/assemble.ts

```diff
@@ -50,6 +50,13 @@ export function sizeSignals(model: Model, sizeType: 'width' | 'height'): NewSign
     }
     /* istanbul ignore next: Condition should not happen -- only for warning in development. */
     throw new Error('layout size is step although width/height is not step.');
+  } else if (size == 'container') {
+    const isWidth = name.endsWith('width');
+    const expr = isWidth ? 'containerSize()[0]' : 'containerSize()[1]';
+    const defaultValue = isWidth ? model.config.view.continuousWidth : model.config.view.continuousHeight;
+    // make sure the value is valid
+    const safeExpr = `isValid(${expr}) ? ${expr} : ${defaultValue}`;
```

Comment:
We use `isFinite` in other places. It should behave the same except that it also filters out infinite values. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite.

I think Vega uses `isFinite` rather than `Number.isFinite` right now but I'll send a PR to change this.

Reply:
Good point! Thanks!